minithesis.py

```diff
@@ -82,16 +82,15 @@
 S = TypeVar("S", covariant=True)
 U = TypeVar("U")  # Invariant
 
+InterestTest = Callable[[array], bool]  # Really array[int] -> bool
```

Comment:
I'd prefer `InterestingnessTest`

Also what's with the comment here? Why can't this be an `array[int]`?

Reply:
The builtin `array` type is not subscriptable, and so the language can't handle `array[int]` here

Reply:
Confirmed works when running on 3.12

Reply:
3.11:
```python
test_minithesis.py:17: in <module>
    import minithesis as mt
minithesis.py:85: in <module>
    InterestTest = Callable[[array[int]], bool]  # Really array[int] -> bool
E   TypeError: type 'array.array' is not subscriptable
```